src/infrastructure/messaging/senders/onebot_sender.py

```
@@ -82,6 +82,34 @@ def _resolve_bot_client(cls, context: MessageContext | None) -> Any | None:
         platform_name = getattr(context, "platform_name", "") if context else ""
         return get_bot_client(platform_name or "")
 
+    @classmethod
+    def _resolve_bot_self_id(
+        cls, context: MessageContext | None, bot_client: Any | None
+    ) -> str:
+        """解析 bot 的 self_id（QQ 号），用于合并转发节点的 user_id。
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)